src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/request.cache.max-age=0.and.304/accept/client.rpt

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newProxyAcceptRef ${nukleus:newRouteRef()} # external scope
+property date ${http_cache:date()} # shared property
+property etag ${http_cache:strongEtag()}
+
+connect await ROUTED_PROXY
+        "nukleus://http-cache/streams/source"
+        option nukleus:route ${newProxyAcceptRef}
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:header(":scheme", "http")}
+                        ${http:header(":method", "GET")}
+                        ${http:header(":path", "/path")}
+                        ${http:header(":authority", "localhost:8080")}
+                        ${http:header("cache-control", "max-age=0")}
+
+
+
+connected
+
+write close
+read nukleus:begin.ext ${http:header(":status", "200")}
+                       ${http:header("content-length", "11")}
+                       ${http:header("cache-control", "max-age=15")}
+                       ${http:header("date", date)}
+                       ${http:header("last-modified", date)}
+
+read "hello world"
+
+read closed
+read notify REQUEST_CACHED
+
+
+# INPUT STREAM 2
+connect await REQUEST_CACHED
+        "nukleus://http-cache/streams/source"
+        option nukleus:route ${newProxyAcceptRef}
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:header(":scheme", "http")}
+                        ${http:header(":method", "GET")}
+                        ${http:header(":path", "/path")}
+                        ${http:header(":authority", "localhost:8080")}
+                        ${http:header("cache-control", "max-age=0")}
+                        ${http:header("etag", etag)}
```

Comment:
Either `last-modified` above and `if-modified-since` here or `etag` above and `if-none-match` here, but not a mixture.

Response below should be `etag` if response above is `etag` or response below should be `last-modified` if response above is `last-modified`, but not a mixture.